manifests/manage_unit.pp

```diff
@@ -142,7 +147,7 @@
     daemon_reload           => $daemon_reload,
     content                 => epp('systemd/unit_file.epp', {
         'unit_entry'    => $unit_entry,
-        'service_entry' => $service_entry,
+        'service_entry' => merge($service_entry, $limit_entry),
```

Comment:
The current `service_entry` , `socket_entry`, ...  parameters correspond to the  `[Service]`, `[Socket]` entries of the unit file.

A `LimitNOFILE` is just a attribute of the existing `[Service]` entry so new parameter for the existing `Systemd::Unit::Service` type ?

I don't believe this should be added like this.

Reply:
Exactly the same as the existing [LimitCORE](https://github.com/voxpupuli/puppet-systemd/blob/master/types/unit/service.pp#L33)

Reply:
If I add all limits to the `Systemd::Unit::Service` it would be possible to set limits with `systemd::manage_dropin` instead of `systemd::service_limits`.
If that is OK I can copy all limits from [Systemd::ServiceLimits](https://github.com/voxpupuli/puppet-systemd/blob/master/types/servicelimits.pp ) to [Systemd::Unit::Service](https://github.com/voxpupuli/puppet-systemd/blob/master/types/unit/service.pp)

Reply:
Could also change the existing `systemd::servicelimits` to just use `systemd::manage_dropin` and drop that duplication. This would be good.

Reply:
Could consider `systemd::servicelimits` for deprecation to be honest. Not as part of this however.

Reply:
@traylenator I added all Limits to `Systemd::Unit::Service`, is this PR now good for merge or do I need to change something else?